cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/dataset_selector.spec.js

```diff
@@ -2,65 +2,50 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-import {
-  MiscUtils,
-  TestFixtureHandler,
-} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
-import { PATHS, SECONDARY_ENGINE } from '../../../../../utils/constants';
 
-const miscUtils = new MiscUtils(cy);
-const testFixtureHandler = new TestFixtureHandler(cy, PATHS.ENGINE);
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import { WORKSPACE_NAME, DATASOURCE_NAME, START_TIME, END_TIME } from './constants';
+import { BASE_PATH, SECONDARY_ENGINE } from '../../../../../utils/constants';
 
-describe.skip('dataset selector', { scrollBehavior: false }, () => {
-  describe('empty state', () => {
-    it('no index pattern', function () {
-      // Go to the Discover page
-      miscUtils.visitPage(
-        `app/data-explorer/discover#/?_g=(filters:!(),time:(from:'2015-09-19T13:31:44.000Z',to:'2015-09-24T01:31:44.000Z'))`
-      );
+const miscUtils = new MiscUtils(cy);
```

Comment:
Do we need the `MiscUtils` since we are only using this to for navigation which we can do using the normal cypress commands.

Reply:
Plan to migrate MiscUtils.visitPage from lib to our cypress in a follow up

Reply:
fix it